product.html

```diff
@@ -22,7 +22,7 @@
             <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
                 <span class="icon-bar"></span>
                 <span class="icon-bar"></span>
-                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>ŃŃ
```

Comment:
to ŃŃ ma być w linijce 25? czy chochlik się wdarł:)

Reply:
Teraz już nie jest potrzebne :)